//! exit_code: 1
fn main() {
    let a = (1, 2, 3);
    a.10;  // diag: error(tuple_index_out_of_bounds): "tuple index out of bounds"
}
